Azure/Azure-Sentinel
# Checking GitHub validations for a Solution

Required items, please complete
   
   Change(s):
   - TESTING the github validations for a solution

   Reason for Change(s):
   - See guidance below

   Version Updated:
   - Required only for Detections/Analytic Rule templates
   - See guidance below

   Testing Completed:
   - See guidance below
